docs/applications/harness-application.md

```diff
@@ -42,4 +44,10 @@ optional arguments:
 Additionally, you can choose one of the following database templates:
 * `db-postgres` - [PostgreSQL](https://www.postgresql.org/), a relational database
 * `db-neo4j`- [Neo4J](https://neo4j.com/), a graph database
-* `db-mongo` - [MongoDB](https://www.mongodb.com/), a NoSQL document-based database
\ No newline at end of file
+* `db-mongo` - [MongoDB](https://www.mongodb.com/), a NoSQL document-based database
+
+### Django
+* It consists of a single backend, a Python [FastAPI](https://fastapi.tiangolo.com/) application. 
+* The [FastAPI code generator](https://github.com/koxudaxi/fastapi-code-generator) maps the OpenAPI definition to FastAPI routing.
+* The [Django framework](https://www.djangoproject.com/) encourages rapid development and clean, pragmatic design.
+* Per default, [Uvicorn](https://www.uvicorn.org/) serves the FastAPI app with 2 workers. Depending on the application requirements, you can update the number of workers.
```

Comment:
Have to check whether the cloudharness debug still works. It's now based on replacing the gunicorn command with something like `python __main__.py` but I'm not sure the detection still works here

Reply:
yes this still works as expected :-)